alb-lambda-rest-api-sam-py/README.md

```diff
@@ -0,0 +1,147 @@
+# AWS Lambda RESTful API with Amazon ALB and Path Based Listener Rules
+
+This configuration employs Python 3.9 and the Serverless Application Model (SAM) CLI to establish an Application Load
+Balancer with path-based listener rules, paired with an AWS Lambda RESTful API function as the target.
+
+Learn more about this pattern
+at [Serverless Land Patterns](https://serverlessland.com/patterns/alb-lambda-rest-api-sam-py).
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free
+Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any
+AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already
+  have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls
+  and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (
+  AWS SAM) installed
+* [Python 3 installed](https://www.python.org/downloads/)
+* [Docker](https://www.docker.com/products/docker-desktop/)
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+2. Change directory to the pattern directory:
+    ```
+    cd alb-lambda-rest-api-sam-py
+    ```
+3. Install dependencies and build:
+    ```
+    pipenv install
+    ```
+4. From the command line, use AWS SAM to deploy the AWS resources for the pattern as specified in the template.yml file:
+    ```
+    sam validate && sam build && sam deploy --guided
+    ```
+5. During the prompts:
```

Comment:
When I run `sam deploy --guided` the  stack also asks me for a VPC id and subnets. Please include instructions what the user needs to do here and what format is expected.